.eslintrc.js

```diff
@@ -15,6 +18,29 @@ module.exports = {
     defineModel: 'readonly'
   },
   rules: {
+    '@intlify/vue-i18n/no-unused-keys': [ 'error', {
+      src: 'src/',
+      callExpression: '^(\\$t|t|\\$tc|tc|\\$tcn|tcn|\\$tn|tn)$',
```

Comment:
Once #1026 is complete, there won't be a `$tc()` or `tc()` anymore. At that time, we'll probably rename `$tcn()` to `$tn()`.

Currently, there is `$tcn()`, and there is `tn()`; there's no `$tn()` or `tcn()`. The asymmetry is because we've had `$tcn()` for a long time, while `tn()` was added around the time that we moved to Vue 3 and started using the Composition API. In the context of Vue I18n within the Composition API, `tc()` is deprecated, because `t()` now does what `$t()` and `$tc()` both used to do.